UI/window-basic-main-transitions.cpp

```diff
@@ -240,6 +240,9 @@ void OBSBasic::TransitionStopped()
 		OBSSource scene = OBSGetStrongRef(swapScene);
 		if (scene)
 			SetCurrentScene(scene);
+
+		/*	Make sure we re-enable the transition button.	*/
```

Comment:
Style:  For a single line comment, using `//` is fine.  Otherwise, if using `/* */`, just use a single space between the comment markers and the contained text.